src/main/java/com/api/dto/Engines.java

```diff
@@ -4,10 +4,12 @@
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 @AllArgsConstructor
 @NoArgsConstructor
 @Getter
+@Setter
 public class Engines {
```

Comment:
where are this object used btw? I think we should have only DTO, SX and JPA models